dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -656,4 +656,8 @@ ssl.client.handshake.cacerts.error.CWPKI0828E.useraction=Review the extended err
 ssl.update.certificate.error=CWPKI0829E: SSL certificate update error. {0}
 ssl.update.certificate.error.explanation=The SSL certificate could not be updated at the specified location.
 ssl.update.certificate.error.useraction=Ensure the location is accessible by the server process.  Review the FFDC logs for additional errors that are associated with generating or loading the keys.
+# -------------------------------------------------------------------------------------------------
+ssl.certificate.add.CWPKI0830I=CWPKI0830I: Certificate with SubjectDN {0} from the environment variable {1} is being added to the {2} keystore.
```

Comment:
**change**
Certificate with SubjectDN {0} from the environment variable {1} is being added to the {2} keystore.
**to**
Certificate with the {0} SubjectDN  from the {1} environment variable  is  added to the {2} keystore.

Reply:
Updated made to the file.   Please check review again.

Reply:
sorry- based on your comment below, should probably be

Certificate with the {0} SubjectDN from the {1} environment variable is being added to the {2} keystore.

Reply:
Okay,  messages has been updated.

Reply:
Just needs an edit so that the variables come before the noun identifiers:

`Certificate with the {0} SubjectDN from the {1} environment variable is being added to the {2} keystore.`

Reply:
Message updated again.  Can you see my comment below about the explanation?

Reply:
Message has been updated please review.